docs/en/tutorials/customize_datasets.md

```diff
@@ -172,3 +172,39 @@ data = dict(
 )
 
 ```
+
+### Multi-image Mix Dataset
+
+We use `MultiImageMixDataset` as a wrapper to mix images from multiple datasets.
+`MultiImageMixDataset` can be used by multiple images mixed data augmentation
+like mosaic and mixup.
+
+An example of using `MultiImageMixDataset` with `Mosaic` data augmentation:
+```python
+train_pipeline = [
+    dict(type='Mosaic'),
```

Comment:
`dict(type='Mosaic')`
-> `dict(type='RandomMosaic', prob=1)`

I think it's better to modify it.